modular_chomp/code/modules/mob/living/simple_mob/subtypes/vore/weather.dm

```diff
@@ -72,90 +201,48 @@
 /mob/living/simple_mob/vore/weatherbeast/sandstorm/do_special_attack(atom/A)
 	if(health < maxHealth*0.25)
 		visible_message(span_orange("<B>The fossil tank begins self repairs!.</B>"))
-		sleep(3)
-		adjustBruteLoss(-40)
-		adjustFireLoss(-40)
-		adjustToxLoss(-40)
-		adjustOxyLoss(-40)
-		adjustCloneLoss(-40)
-		sleep(1)
-		adjustBruteLoss(-40)
-		adjustFireLoss(-40)
-		adjustToxLoss(-40)
-		adjustOxyLoss(-40)
-		adjustCloneLoss(-40)
-		sleep(1)
-		adjustBruteLoss(-40)
-		adjustFireLoss(-40)
-		adjustToxLoss(-40)
-		adjustOxyLoss(-40)
-		adjustCloneLoss(-40)
-		sleep(1)
-		adjustBruteLoss(-40)
-		adjustFireLoss(-40)
-		adjustToxLoss(-40)
-		adjustOxyLoss(-40)
-		adjustCloneLoss(-40)
-		sleep(1)
-		adjustBruteLoss(-40)
-		adjustFireLoss(-40)
-		adjustToxLoss(-40)
-		adjustOxyLoss(-40)
-		adjustCloneLoss(-40)
-		sleep(1)
-		adjustBruteLoss(-40)
-		adjustFireLoss(-40)
-		adjustToxLoss(-40)
-		adjustOxyLoss(-40)
-		adjustCloneLoss(-40)
-		sleep(1)
-		adjustBruteLoss(-40)
-		adjustFireLoss(-40)
-		adjustToxLoss(-40)
-		adjustOxyLoss(-40)
-		adjustCloneLoss(-40)
-		sleep(1)
-		adjustBruteLoss(-40)
-		adjustFireLoss(-40)
-		adjustToxLoss(-40)
-		adjustOxyLoss(-40)
-		adjustCloneLoss(-40)
-		sleep(1)
-		adjustBruteLoss(-40)
-		adjustFireLoss(-40)
-		adjustToxLoss(-40)
-		adjustOxyLoss(-40)
-		adjustCloneLoss(-40)
-		sleep(1)
-		adjustBruteLoss(-40)
-		adjustFireLoss(-40)
-		adjustToxLoss(-40)
-		adjustOxyLoss(-40)
-		adjustCloneLoss(-40)
-		sleep(1)
-		adjustBruteLoss(-40)
-		adjustFireLoss(-40)
-		adjustToxLoss(-40)
-		adjustOxyLoss(-40)
-		adjustCloneLoss(-40)
-		sleep(1)
-		adjustBruteLoss(-40)
-		adjustFireLoss(-40)
-		adjustToxLoss(-40)
-		adjustOxyLoss(-40)
-		adjustCloneLoss(-40)
+		sleep(30)
```

Comment:
why isn't this in a loop?

Reply:
Unsure how to create one, and what purpose that would serve of it being in a loop

Reply:
Guessing it's length of code. Altered it to just have a longer cooldown instead of repeated healing overtime alongside larger healing on the outset.
I do apologize if the previous comment came off as harsh, meant to be a question mark

Reply:
I'll push an example, it's usually better to loop code than to copy paste it:

for(var/i =1 to 6)
    do_something()

the loop will now recall the same function. So instead of copypasting the code very often, you just increase the to value to have it loop more often

Reply:
For your code it would be something like

```
for(var/i = 1 to 8)
	adjustBruteLoss(-80)
	adjustFireLoss(-80)
	sleep(10)
adjustBruteLoss(-80)
adjustFireLoss(-80)
```

We don't want to sleep after the final proc again

Reply:
Oh that is oddly simple. Thank you. Altered the tank code to utilize this, and for another example of it later on.